src/utils/addressFormat/cashAddress.js

```diff
@@ -1,12 +1,10 @@
 // @flow
-
+// $FlowFixMe
```

Comment:
This `$FlowFixMe` is probably not needed, since you were able to delete all the other ones like this.

Reply:
Unfortunately it is still needed because of the buffer-hack :(
I'll get it fixed eventually but it will needs a change to the build system to work.